### ISSUE TITLE
wails init fails for Vue3 Full

### ISSUE THREAD

**Issue by sascha-andres** [wailsapp/wails#574]
**Description**

When using the template for Vue3 Full npm i fails because it cannot resolve the dependency tree. my_project required eslint@7.15.0 while @vue/cli-plugin-eslint requires >= 1.6.0 < 7.0.0

**To Reproduce**
Steps to reproduce the behaviour:
1. wails init
2. Choose 5 Vue3 Full
3. See error

**Expected behaviour**
A running templated wails application

**Screenshots**

```Wails v1.10.1 - Initialising project

The name of the project (My Project):
Project Name: My Project
The output binary name (my-project):
Output binary Name: my-project
Project directory name (my-project):
Project Directory: my-project
Please select a template (* means unsupported on current platform):
  1: Angular - Angular 8 template (Requires node 10.8+)
  2: React JS - Create React App v4 template
  3: Svelte - A basic Svelte template
  4: Vanilla - A Vanilla HTML/JS template
  5: Vue3 Full - Vue 3, Vuex, Vue-router, and Webpack4
  6: Vue2/Webpack Basic - A basic Vue2/WebPack4 template
  7: Vuetify1.5/Webpack Basic - A basic Vuetify1.5/Webpack4 template
  8: Vuetify2/Webpack Basic - A basic Vuetify2/Webpack4 template
Please choose an option [1]: 5
Template: Vue3 Full
✓ Generating project...
⣻ Building project (this may take a while)...
Wails v1.10.1 - Building Application

⣻ Ensuring frontend dependencies are up to date (This may take a while)npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my_project@undefined
npm ERR! Found: eslint@7.15.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^7.10.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">= 1.6.0 < 7.0.0" from @vue/cli-plugin-eslint@4.5.9
npm ERR! node_modules/@vue/cli-plugin-eslint
npm ERR!   dev @vue/cli-plugin-eslint@"~4.5.6" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /home/andres/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/andres/.npm/_logs/2020-12-12T15_59_07_718Z-debug.log


✗ Ensuring frontend dependencies are up to date (This may take a while)
Error: exit status 1

✗ exit status 1
Error: exit status 1```

**System Details**

| Name   | Value |
| ----- | ----- |
| Wails Version | v1.10.1 |
| Go Version    | go1.15.6 |
| Platform      | linux |
| Arch          | amd64 |
| GO111MODULE   | (Not Set) |
| GCC           | 10.2.0 |
| Npm           | 7.0.15 |
| Node          | v15.4.0 |

**Comment by sascha-andres** [comment]
I found and read #567 but wails update did not resolved that issue

**Comment by leaanthony** [comment]
Hi. As it's an npm versioning issue related to your project, I'm not really sure what you want us to do? Have you tried downgrading your eslint?

**Comment by sascha-andres** [comment]
As this happens directly after wails init this is an issue with the template. I just assumed that each template is designed to work out of the box (like the react one does).

**Comment by leaanthony** [comment]
Ah, if updating works, would be happy to accept a PR 👍

**Comment by leaanthony** [comment]
This is now fixed in `v1.11.0-pre1`. `wails update -pre` to get it!

**Comment by sascha-andres** [comment]
@leaanthony  Can confirm this! Thanks. Closing
